chrome/entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+if [ "$(ls -A /home/chrome/.fonts/)" ]; then
+  fc-cache -f -v
+fi
+
+(ulimit -n 65000 || true) && (ulimit -p 65000 || true) && google-chrome-stable \
+  --enable-automation \
+  --disable-background-networking \
+  --disable-background-timer-throttling \
+  --disable-backgrounding-occluded-windows \
+  --disable-renderer-backgrounding \
+  --disable-breakpad \
+  --disable-client-side-phishing-detection \
+  --disable-default-apps \
+  --disable-dev-shm-usage \
+  --disable-extensions \
+  --disable-gpu \
+  --disable-popup-blocking \
+  --disable-prompt-on-repost \
+  --disable-sync \
+  --disable-translate \
+  --disable-ipc-flooding-protection \
+  --disable-component-update \
+  --headless \
```

Comment:
it should be `--headless=chrome`

Reply:
Should I add the `--site-per-process` too here?

Reply:
@shevchenkov-neuralegion nope, you shouldn't